### DISCUSSION TITLE
Does the Authenticated component cover all {children}?

### DISCUSSION THREAD

**Discussion by marqpdx** [refinedev/refine#4933] [Q&A]
Can i just wrap the top-level layout in the `<Authenticated/>` component (using in my case Nextjs w/ App) and have all child pages thereby inherit the requirement to authenticate, or must i wrap each page in its own Authenticated component?
Thanks!

**Comment by BatuhanW** [comment]
Hello @marqpdx, if you wrap top-level component with `Authenticated` component, it will work for client-side, however, if you want to have server-side validation, you can read the following documentation.

https://refine.dev/docs/packages/documentation/routers/nextjs/#authentication
